apps/api/src/main.mts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2024 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import cors from 'cors';
+import express, { Request, Response } from 'express';
+
+// TODO: fix import
+import { demoApplication } from 'pcgl-daco/packages/types/src/main.mjs';
+
+const app = express();
+
+// Exports only to demo test suite
+export const port = 3000;
+
+app.use(cors());
+
+app.get('/', (req: Request, res: Response) => {
+	console.log(req);
+	res.send('Hello World!');
+});
+
+app.get('/applications', cors(), (req: Request, res: Response) => {
+	// TODO: fix unused req
```

Comment:
Try `_req`. The underscore prefix indicates this is a placeholder parameter and is unused. TS will be happy with that. This TODO is why it may not make sense to enforce no unused params/locals.

Reply:
Thank you did not know about that trick